src/wp-includes/ms-blogs.php

```diff
@@ -666,7 +666,7 @@ function restore_current_blog() {
 				);
 			}
 
-			wp_cache_add_non_persistent_groups( array( 'counts', 'plugins' ) );
+			wp_cache_add_non_persistent_groups( array( 'plugins' ) );
```

Comment:
I tend to agree with @spacedmonkey's comment in Slack that this probably can't be done due to the risk plugins are using the group assuming it's non-persistent.

Reply:
I reluctantly agree, as such, I've moved these calls to use a new cache group `persistent-counts` instead.

Although the `comment` group was moved from non-persistent to persistent, it seems like it's less often used than the `counts` group in plugins.